### DISCUSSION TITLE
支付宝小程序无法播放模型里面的动画

### DISCUSSION THREAD

**Discussion by baron517** [galacean/discussion#9] [Mini Program（小程序）]
const animator = this.gltf.defaultSceneRoot.getComponent(Animator);
    console.log("动画",animator,this.gltf.animations[0].name);
网页里面是好的 小程序 animator打出来是undefined 同一模型，引擎版本一直 都是0.8 具体复现请看我的小程序代码链接地址  https://github.com/baron517/oasis-demo 直接支付宝小程序开发工具打开就可以看到效果

**Comment by GuoLei1990** [comment]
版本过旧，可以尝试升级到最新的 1.2 版本，如果仍有问题可重新打开讨论，我们会跟进
